wallet/build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
     implementation "org.dashj:dashj-core:$dashjVersion"
     implementation 'com.google.guava:guava:29.0-android'
-    implementation 'com.google.zxing:core:3.4.1'
+    implementation 'com.google.zxing:core:3.3.3'
```

Comment:
Let's add a comment here saying that 3.3.3 is the maximum to support Android 4.4 to 6.x

Otherwise, we may forget the next time we want to update libraries.

Reply:
Good idea!
Done